app/__init__.py

```diff
@@ -31,16 +32,22 @@ def create_app(config_name):
         search_api_client=search_api_client
     )
 
-    for framework_data in data_api_client.find_frameworks().get('frameworks'):
+    frameworks = data_api_client.find_frameworks().get('frameworks')
+    for framework_data in frameworks:
         if not framework_data['slug'] in application.config.get('DM_FRAMEWORK_CONTENT_MAP', {}):
             if framework_data['framework'] == 'g-cloud':
                 if framework_data['status'] != 'expired':
-                    content_loader.load_manifest(framework_data['slug'], 'services', 'search_filters')
+                    content_loader.load_manifest(framework_data['slug'], 'services', 'services_search_filters')
                 # we need to be able to display old services, even on expired frameworks
                 content_loader.load_manifest(framework_data['slug'], 'services', 'display_service')
                 content_loader.load_manifest(framework_data['slug'], 'services', 'download_results')
             elif framework_data['framework'] == 'digital-outcomes-and-specialists':
                 content_loader.load_manifest(framework_data['slug'], 'briefs', 'display_brief')
+                content_loader.load_manifest(
+                    get_latest_live_framework(frameworks, 'digital-outcomes-and-specialists')['slug'],
```

Comment:
We're iterating through frameworks, so this actually means we're going to load the DOS2 filters manifest _twice_.

Maybe we could split loading the search filters manifests into a separate loop (over framework families), covering both DOS and G-Cloud.

Reply:
Sugar. Yep, I'll do the separate loop. It makes sense if we'll only have one _something_search_filters manifest per family.

Reply:
Have actually just pulled out the loading of the DOS2 manifest to a separate thing. Looping over framework families didn't really work, as the criteria for which one to load is different depending on the framework family (g-cloud, if the framework isn't expired, dos, the latest framework iteration)